manifests/p/PrismLauncher/PrismLauncher/7.0/PrismLauncher.PrismLauncher.installer.yaml

```diff
@@ -0,0 +1,33 @@
+# Created with WinGet Releaser v2 using Komac v1.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: PrismLauncher.PrismLauncher
+PackageVersion: '7.0'
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2023-06-08
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/PrismLauncher/PrismLauncher/releases/download/7.0/PrismLauncher-Windows-MSVC-Legacy-Setup-7.0.exe
+  InstallerSha256: 2C012ECF45B89E0D125A8317C6251ECEB12865B6D0C640BA5A2772EC71C7E3B7
+  Dependencies:
+    PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
+- Architecture: x64
+  InstallerUrl: https://github.com/PrismLauncher/PrismLauncher/releases/download/7.0/PrismLauncher-Windows-MSVC-arm64-Setup-7.0.exe

```

Comment:
The download link for x64 is wrong, shouldn't it be `https://github.com/PrismLauncher/PrismLauncher/releases/download/7.0/PrismLauncher-Windows-MSVC-Setup-7.0.exe`?

Reply:
That is indeed wrong. I guess WinGet Releaser failed to properly detect these executables. Let me fix this